smarts/env/wrappers/single_agent.py

```diff
@@ -58,7 +58,7 @@ def step(self, action: Any) -> Tuple[Any, float, bool, Any]:
         return (
             obs[self._agent_id],
             reward[self._agent_id],
-            done[self._agent_id],
+            done,
```

Comment:
I think this should not be changed, as it is intended to wrap a multi-agent env to become compatible with a single-agent gym specs.
```suggestion
            done[self._agent_id],
```

Reply:
See other comment